etc/tendrl/alerting.conf.yaml.sample

```diff
@@ -0,0 +1,9 @@
+---
+# Central store etcd port
+etcd_port: 2379
+# Central store etcd host/ip
+etcd_connection: 0.0.0.0
+log_cfg_path: /etc/tendrl/alerting/alerting_logging.yaml
+log_level: DEBUG
+tendrl_ansible_exec_file: /root/.tendrl/alerting/ansible_exec
+logging_socket_path: /var/run/tendrl/message.sock
```

Comment:
@r0h4n socket name is message.sock only, we have to change in gluster-integration